docs/pudl_overview.rst

```diff
@@ -0,0 +1,132 @@
+=======================================================================================
+PUDL Overview
+=======================================================================================
+
+PUDL is a data processing pipeline that cleans, connects, and standardizes data from
+some of the most widely used public energy datasets. The data serve researchers,
+activists, journalists, and policy makers that might not have the means to purchase
+processed data from existing commercial providers, the technical expertise to
+access it as published, or the time to clean and prepare the data for bulk analysis.
+
+Currently, PUDL processes, integrates, and provides enhanced access to data from EIA
+Forms 860 and 923, FERC Forms 1 and 714, and EPA CEMS. See our `website <https://catalyst.coop/>`_
+for more data in the works!
+
+If you want to get started using PUDL data, visit our :doc:`usage_modes` page. Read on
+to learn about the components of the data processing pipeline.
+
+.. _raw-data-archive:
+
+---------------------------------------------------------------------------------------
+Raw Data Archive
+---------------------------------------------------------------------------------------
+
+Scripts that reliably extract data published online require consistent access to
```

Comment:
This might be clearer if it was "In order for scripts that extract data published online to be reliable, they need consistent access to consistent data."